src/main/java/com/softserveinc/ita/pageobjects/admin/ProtocolPage.java

```diff
@@ -1,4 +1,48 @@
 package com.softserveinc.ita.pageobjects.admin;
 
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$x;
+import static java.lang.String.format;
+
 public class ProtocolPage extends MainMenu {
+
+    private final SelenideElement startDatePickerButton = $x("(//button[@aria-label='Open calendar'])[1]");
+    private final SelenideElement endDatePickerButton = $x("(//button[@aria-label='Open calendar'])[2]");
+    private final SelenideElement chooseMonthAndYearButton = $x("//button[@aria-label='Choose month and year']");
+    private final SelenideElement searchButton = $x("//button[@type='submit']");
+
+    private final String optionButtonTemplate = "//div[text()='%s']";
+
+    public ProtocolPage chooseStartDate(String startYear, String startMonth, String startDay) {
```

Comment:
use LocalDateTime class from org.joda.time lib, and create a class in 'models' package or somewhere else

```
@Builder
@Data
public class DateTimeRange {
    private LocalDateTime start;
    private LocalDateTime end;
}
```

and you would be able to pass only one param into the method (but may not be required within this task)
```
var startDate = LocalDateTime.parse("2016-02-13", forPattern("YYYY-MM-dd"));
var endDate = LocalDateTime.now();
var someRange = DateTimeRange
.builder()
.start(startDate)
.end(endDate)
.build();
```

also, you could create a private method and move some things from your methods:

```
private ProtocolPage chooseDate(LocalDateTime date, int index) {
        $x(format(DATE_PICKER_TEMPLATE, index)).click();
        chooseMonthAndYearButton.click();
        $x(format(optionButtonTemplate, date.getYear())).click();
        $x(format(optionButtonTemplate, date.getMonthOfYear())).click();
        $x(format(optionButtonTemplate, date.getDayOfMonth())).click();

        return this;
    }
```

```
public ProtocolPage chooseStartDate(LocalDateTime date) {
return chooseDate(date, 1);
}

public ProtocolPage chooseEndDate(LocalDateTime date) {
return chooseDate(date, 2);
}
```